tests/integration/query/query_test.go

```diff
@@ -36,6 +36,7 @@ func TestQuery(t *testing.T) {
 		Dir:          "step1",
 		Dependencies: []string{"@pulumi/kubernetes"},
 		Quick:        true,
+		StackName:    "query-test", // Have to hardcode stack name because query does not run as part of a stack.
```

Comment:
I think that this will cause failures if this test runs concurrently with itself (or others using the same name, see e.g. https://travis-ci.com/pulumi/pulumi/jobs/251756294). We should mix in some randomness.